src/client/pages/Offer/Introduction/Sidebar/CancellationOptions.tsx

```diff
@@ -124,13 +120,9 @@ const QuoteCancellationOption: React.FC<QuoteCancellationOptionProps> = ({
             return textKeys.SIDEBAR_REQUEST_CANCELLATION_GENERIC_INSURANCE()
           }
 
-          if (isNorwegianHomeContents(quote.quoteDetails)) {
-            return textKeys.SIDEBAR_REQUEST_CANCELLATION_HOME_INSURANCE()
-          }
-
-          if (isNorwegianTravel(quote.quoteDetails)) {
-            return textKeys.SIDEBAR_REQUEST_CANCELLATION_TRAVEL_INSURANCE()
-          }
```

Comment:
Do we not need these checks anymore?

Reply:
I'm glad you've pointed this out

This is an attempt of moving away from code that rely on quote's type as a consequence of a more generic approach with `quote.data`. But ofc I would like to discuss it more as I don't fully understand the business logic here.

In a scenario of Swedish market and multiple quotes ([isGenericQuote](https://github.com/HedvigInsurance/web-onboarding/pull/750/files#diff-7c371857b44f7fecfd1b97f2c85ceea91357515d3c034c39faaba60335e4156eL13) is based on the amount of quotes) we'd end up rendering the Checkbox with no label. But it looks like [switchable](https://github.com/HedvigInsurance/web-onboarding/pull/750/files#diff-7c371857b44f7fecfd1b97f2c85ceea91357515d3c034c39faaba60335e4156eL13) is always false on such cases.

Couldn't we assume `switchable` flag will take also market into account and display cancellation options as following?

If only one quote (`isGenericQuote === true`), then display one cancellation option with generic wording (`textKeys.SIDEBAR_REQUEST_CANCELLATION_GENERIC_INSURANCE()`)
else display, for each quote, a cancellation option that mentions the quote name.

Reply:
Aha, so you are proposing essentially making this component market-agnostic? At least in terms of the labels, right?

Reply:
Yes